xml/System.Reflection.PortableExecutable/PdbChecksumDebugDirectoryData.xml

```diff
@@ -63,7 +63,7 @@
         <ReturnType>System.Collections.Immutable.ImmutableArray&lt;System.Byte&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>GUID (Globally Unique Identifier) of the associated PDB.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>The GUID of the associated PDB.</value>
```

Reply:
This description was wrong as mentioned in the other comment. I updated it.